AccuracyIndicator/Indicator/InGameIndicator.cs

```diff
@@ -39,11 +39,11 @@ private void Start()
         _canvas.AddComponent<GraphicRaycaster>();
         _canvas.GetComponent<Canvas>().renderMode = RenderMode.ScreenSpaceOverlay;
 
-        // -0.14 | -0.05 | -0.025 | 0.025 | 0.05 | 0.14
+        // -0.13 | -0.05 | -0.025 | 0.025 | 0.05 | 0.13
```

Comment:
Not sure if we really should restrict to 0.13, I managed to get -0.135 (by smashing my keyboard, but I reached this value)
But if you really want it, you should also update the bar widths to fit (and the position of the hit markers)

Reply:
well from the aspect that the game set it as 0.13 ig we should use it? to make it more accurate from the game side

Reply:
Okay! Then we'll need to update the widths of the different things. Would you like to do it or do I?

Reply:
yeah i didnt look detailed about these code (also not that understand bc my low level) so i think it's better for you to do this 👍